site/content/en/guides/producer/functions/ts/quickstart/_index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "TypeScript Quickstart"
+linkTitle: "TypeScript Quickstart"
+weight: 6
+type: docs
+description: >
+   Quickstart for developing Typescript functions.
+---
+
+## Developer Quickstart
+
+This quickstart will get you started developing a config function with the TypeScript SDK,
+using an existing Hello World package.
+
+### Prerequisites
+
+#### System Requirements
+
+Currently supported platforms: amd64 Linux/Mac
+
+#### Setting Up Your Local Environment
+
+- Install [node][download-node]
+- Install [docker][install-docker]
+- Install [kpt][download-kpt] and add it to \$PATH
+
+### Hello World Package
+
+1. Get the `hello-world` package:
+
+   ```sh
+   git clone --depth 1 https://github.com/GoogleContainerTools/kpt-functions-sdk.git
+   ```
+
+   All subsequent commands are run from the `hello-world` directory:
+
+   ```sh
+   cd kpt-functions-sdk/ts/hello-world
+   ```
+
+1. Install all dependencies:
+
+   ```sh
+   npm install
+   ```
+
+1. Run the following in a separate terminal to continuously build your function as you make changes:
+
+   ```sh
+   npm run watch
+   ```
+
+1. Run the [`label_namespace`][label-namespace] transformer function:
```

Comment:
Split this into 2 steps (Each step should have one call to action. e.g. "explore" or "run", etc.)

1. Explore label_namespace.
2. Run label_namespace

Reply:
Done for all functions: label_namespace, validate_rolebinding, and expand_team_cr